bridge/discord/discord.go

```diff
@@ -364,6 +382,34 @@ func (b *Bdiscord) replaceChannelMentions(text string) string {
 	return text
 }
 
+func (b *Bdiscord) replaceUserMentions(text string) string {
+	re := regexp.MustCompile("@[^@]{1,32}")
+	text = re.ReplaceAllStringFunc(text, func(m string) string {
+		mention := strings.TrimSpace(m[1:])
+		var member *discordgo.Member
+		var err error
+		for {
```

Comment:
I don't think we need the for loop here? Also the `var member` and `var err` can be removed.
something like below should work (not tested)
```
        b.Log.Debugf("Testing mention: '%s'", mention)
	member, err := b.getGuildMemberByNick(mention)
	if err != nil {
		lastSpace := strings.LastIndex(mention, " ")
		if lastSpace != -1 {
			mention = strings.TrimSpace(mention[0:lastSpace])
		}
                return m
        }
	return member.User.Mention()
```